docs/changelog-fragments.d/526.feature.md

```diff
@@ -0,0 +1,8 @@
+Added the ability to run ad-hoc commands directly within an execution
+environment using mode `stdout`.
+
+```
+$ ansible-navigator exec --mode stdout
```

Comment:
There are a few things to note here:
1) It's usually useful to select the highlighting language for code blocks to avoid ambiguity. This has a better chance of being parsed with the desired lexer. For snippets of commands having `$` prompt, it should usually be `console` or its alias `shell-session`.
2) The way `console` works is that it uses different colors for the commands with prompts and their output.
3) markdownlint will probably be unhappy about having a prompt+command with no output (I've hit that rule recently). Although, you won't see this problem until the release (when this gets into `CHANGELOG.md`) because the notes are excluded from linting.
4) If you only intend to demonstrate a command, you could use `bash`/`sh` instead and drop the prompt.
````suggestion
```console
$ ansible-navigator exec --mode stdout
[example output]
````

Reply:
I'm gonna leave it without a prompt or the output of the command.  The reason I added this wan't really so it showed the sample output, but to show how the command looked with `exec` and the ability to cut-paste from the change log...  My theory, completely unproven, is that someone might be more likely to try it out if they could cut and paste :)

Reply:
> My theory, completely unproven, is that someone might be more likely to try it out if they could cut and paste :) (or use the clipboard icon when it's available)

AFAIR that clipboard icon Sphinx extension actually removes the prompt on copy and it's configurable what it should remove. Needs some testing, though, because I'm not sure how it's configured currently.